purchase_line_product_image/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2017
```

Comment:
Your copyright should be one line. It's also worth noting that in a lot of countries, the symbol isn't a valid claim of copyright. I recommend using only the word - `Copyright 2017 My Company`